docs/deploy-worker-udf-binaries.md

```diff
@@ -0,0 +1,101 @@
+# Deploy Worker and UDF Binaries General Instruction
+
+This documentation will describe how to deploy Worker and UDF (User Defined Function) binaries, which Environment Variable to set up and some commonly used options when launching applications with `spark-submit`.
+
+## Table of Contents
+- [Configurations](#configurations)
+  - [Environment Variables](#1-environment-variables)
+  - [Parameter Options](#2-parameter-options)
+- [Frequently Asked Questions](#faq)
+
+## Configurations
+
+### 1. Environment Variables
+Please see the commonly used environment variables as below when deploying worker and writing udfs. 
+
+<table>
+  <tr>
+    <td width="25%"><b>Environment Variable</b></td>
+    <td width="75%"><b>Description</b></td>
+  </tr>
+  <tr>
+    <td><b>DOTNET_WORKER_DIR</b></td>
+    <td>Path where the <code>Microsoft.Spark.Worker</code> binary has been generated.</br>It's used by the Spark driver and will be passed to Spark executors. If this variable is not set up, the Spark executors will search the path specified in the <code>PATH</code> environment variable.</br><i>e.g. "C:\bin\Microsoft.Spark.Worker-0.6.0"</i></td>
+  </tr>
+  <tr>
+    <td><b>DOTNET_ASSEMBLY_SEARCH_PATHS</b></td>
+    <td>Comma-separated paths where <code>Microsoft.Spark.Worker</code> will load assemblies.</br>Note that if a path starts with ".", the working directory will be prepended. If in <b>yarn mode</b>, "." would represent the container's working directory.</br><i>e.g. "C:\Users\&lt;user name&gt;\&lt;mysparkapp&gt;\bin\Debug\&lt;dotnet version&gt;"</i></td>
+  </tr>
+  <tr>
+    <td><b>DOTNET_WORKER_DEBUG</b></td>
+    <td>If you want to <a href="https://github.com/dotnet/spark/blob/master/docs/developer-guide.md#debugging-user-defined-function-udf">debug UDF</a>, then set this environmnet variable to <code>1</code> before running <code>spark-submit</code>.</td>
+  </tr>
+</table>
+
+### 2. Parameter Options
+Once the spark application is bundled, you can launch it using `spark-submit`. The following table shows some of the commonly used options. 
+
+<table>
+  <tr>
+    <td width="25%"><b>Parameter Name</b></td>
+    <td width="75%"><b>Description</b></td>
+  </tr>
+  <tr>
+    <td><b>--class</b></td>
+      <td>The entry point for your application <i>e.g. org.apache.spark.deploy.dotnet.DotnetRunner</i></td>
```

Comment:
```suggestion
      <td>The entry point for your application </br><i>e.g. org.apache.spark.deploy.dotnet.DotnetRunner</i></td>
```

Consider putting the examples on their own line for consistency/potentially making it a bit easier to read.